src/components/UserSelector.tsx

```diff
@@ -73,10 +70,10 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
```

Comment:
The `href` attribute in the anchor tags is used for navigation but might not be necessary if these links are not intended to navigate to a different page. Consider removing it if it's not needed.